dart/lib/src/platform_checker.dart

```diff
@@ -1,7 +1,13 @@
+import 'platform/platform.dart';
+import 'utils.dart' as utils;
+
 /// Helper to check in which enviroment the library is running.
 /// The envirment checks (release/debug/profile) are mutually exclusive.
 class PlatformChecker {
-  const PlatformChecker();
+  PlatformChecker({
+    this.platform = instance,
+    this.isWeb = utils.isWeb,
```

Comment:
maybe we delete `utils.isWeb` and make the check here only and reuse PlatformChecker? guess it makes more sense